src/Microsoft.ML.Tokenizers/Tokenizer.cs

```diff
@@ -527,6 +527,22 @@ public static Tokenizer CreateTiktokenForEncoding(string encodingName, IReadOnly
             bool addEndOfSentence = false)
             => CreateCodeGen(vocabStream, mergesStream, addPrefixSpace, addBeginOfSentence, addEndOfSentence);
 
+        public static CodeGen CreatePhi2(
+            string folder,
```

Comment:
I am not getting why you need this? can't you do that in the calling site in a helper method? Any API we expose need to be carefully evaluated before exposing it. Also, I am planning to expose something for Ph3 when fixing the special tokens issue. 

FYI, I am currently refactoring a lot of stuff in the code which we'll visit all calling sites and update them.

Reply:
I thought this is the place  where I can add tokenizer helper method to? If not I can move it towards `phi` project.

Reply:
Please move it to your consuming project. Thanks!

Reply:
Removed